pkg/chartvalues/apiextensions_release_e2e_test.go

```diff
@@ -8,12 +8,18 @@ import (
 
 func newAPIExtensionsReleaseConfigFromFilled(modifyFunc func(*APIExtensionsReleaseE2EConfig)) APIExtensionsReleaseE2EConfig {
 	c := APIExtensionsReleaseE2EConfig{
-		Namespace: "default",
-
-		Operator: APIExtensionsReleaseE2EConfigOperator{
-			Name:    "test-operator",
-			Version: "1.0.0",
+		Active: true,
+		Authorities: []APIExtensionsReleaseE2EConfigAuthority{
+			{
+				Name:    "test-operator",
+				Version: "1.0.0",
+			},
 		},
+		Date:      "0001-01-01T00:00:00Z",
+		Name:      "1.0.0",
+		Namespace: "default",
+		Provider:  "AWS",
```

Comment:
So far the provider was lower case. Would be pretty nice to get admission controllers into place and validate such things.